scripts/runtests.py

```diff
@@ -44,11 +46,21 @@ def _get_strict_params(stub_path: str) -> list[str]:
 
 
 def main() -> None:
-    try:
-        path = sys.argv[1]
-    except IndexError:
-        print("Missing path argument in format <folder>/<stub>", file=sys.stderr)
-        sys.exit(1)
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--run-stubtest",
+        type=str.lower,
+        choices=_RUN_STUBTEST_CHOICES,
+        default=None,
+        help="Run or skip stubtest without prompt. Only use this if you trust the package you are testing.",
+    )
```

Comment:
I think it would be more intuitive if we just had this as a simple `store_true` argument, and deleted the `input()` prompt section of the script on lines 114-122:

```suggestion
    parser.add_argument(
        "--run-stubtest",
        action="store_true",
        help="Run stubtest for the selected package(s). Running stubtest may download and execute arbitrary code from PyPI: only use this option if you trust the package you are testing.",
    )
```

Then you could just do `python scripts/runtests.py stubs/requests --run-stubtest`, rather than  `python scripts/runtests.py stubs/requests --run-stubtest yes`, which feels tedious.

Reply:
![image](https://user-images.githubusercontent.com/1350584/219964466-c872f6c3-4f03-43d3-b746-404a66a0d32c.png)